pkg/replicatedapp/api.go

```diff
@@ -149,6 +180,14 @@ func getApplicationMetadataFromHost(host string, endpoint string, upstream *url.
 		return nil, errors.Wrap(err, "failed to call newrequest")
 	}
 
+	appID, err := store.GetStore().GetAppIDFromSlug(r.AppSlug)
```

Comment:
It looks like `store.GetStore()` is returning `nil` here as well based on the failing e2e jobs. This function is called in the context of the KOTS CLI, which is why it does not have direct access to the store.

Reply:
So `store.GetStore()`, might not be returning `nil` here as I had thought above, but this would get called at a time where rqlite has not yet been deployed by KOTS and is the cause of the failing tests. Looking at the CI jobs, the `kots install` fails before any of the components are deployed. My suggestion would be to remove reporting from the metadata request here since it happens before a license is even installed. This initial metadata request is unauthenticated and is just used to apply branding to the initial screens.